### ISSUE TITLE
use modSettings to make keybind for open/close inv window

### ISSUE THREAD

**Issue by rempAut** [rempAut/PZ-ClothingUI#36]

**Comment by R3ality** [comment]
Good idea. When this is implemented perhaps the INV button UI element could be toggleable in modSettings to control if it is visible or not.

The game can already be visually very cluttered in terms of UI. It might be preferable if the equipped items window could be opened by keypress or a right-click menu option on your character instead of a visual button on the screen. Having it configurable via settings would be the best of both worlds and allow the player to choose.

**Comment by petrvecera** [comment]
Done in b42
